api/v1/synthesizer.go

```diff
@@ -16,6 +16,28 @@ type SynthesizerList struct {
 // Synthesizer processes are given some metadata about the composition they are synthesizing, and are expected
 // to return a set of Kubernetes resources. Essentially they generate the desired state for a set of Kubernetes resources.
 //
+// A special resource can be returned from synthesizers: `eno.azure.io/v1.Patch`.
```

Comment:
We might need to add typing for this because for the synthesizer to return a correctly serialized Patch as unstructured, we need it in the schema

Reply:
Hmm yeah it would be helpful to expose a scheme that has this type. Mind if that's a follow up? I have a couple of bug fixes that are going to be easier to land if we merge this first 😢

Reply:
No problem with a follow up :)